build/code_generation_test.bzl

```diff
@@ -32,8 +32,8 @@ def _go_pkg_test_impl(ctx):
     env = unittest.begin(ctx)
     test_cases = [
         ("pkg/kubectl/util", "k8s.io/kubernetes/pkg/kubectl/util"),
-        ("vendor/some/third/party", "k8s.io/kubernetes/vendor/some/third/party"),
-        ("staging/src/k8s.io/apimachinery/api", "k8s.io/kubernetes/vendor/k8s.io/apimachinery/api"),
+        ("vendor/some/third/party", "some/third/party"),
+        ("staging/src/k8s.io/apimachinery/api", "k8s.io/apimachinery/api"),
```

Comment:
Changes related to gopath are expected. We fixed go_genrule in https://github.com/kubernetes/repo-infra/pull/122 to set things up properly.